searches/publicQuery.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const queryResource = require('../resources/query');
```

Comment:
What's the difference between this file and `searches/orgQuery.js`?

Reply:
The only difference is publicQuery.js sets up a search action on the app that only searches coveo help documents (coveosearch org). I felt like that process should be it's own action, since if a user wants to control the sorting of the documents they search in their own organization they can, whereas the search to Coveo public docs we want to control the sorting. It also would get rid of some confusion for users using the Zap, since if they only want help docs from Coveo they can simply just pick this action instead of orgQuery.js and read the help texts. I can easily combine orgQuery and publicQuery if you think this is a bad idea.